docs/manual/manual.adoc

```diff
@@ -1,6 +1,7 @@
 = OpenSCAP User Manual
 :imagesdir: ./images
 :workbench_url: https://www.open-scap.org/tools/scap-workbench/
+:json_tailoring_url: https://github.com/ComplianceAsCode/schemas/tree/main/tailoring
```

Comment:
@evgenyz It would be great if the directory https://github.com/ComplianceAsCode/schemas/tree/main/tailoring contains a README with a short description :smiley:

Reply:
The schema is somewhat documented within the schema file itself. Anyway, it is a different repo with its own issues and PRs.